src/App.js

```diff
@@ -17,7 +17,7 @@ const App = () => {
        <li><Link to ="/pizza">Order</Link></li>
    </ul>
 </nav>
-</Header>
+</header>
 
  <Switch>
  <Route exact path = "/" component={Home} />
```

Comment:
In newer versions of React Router, you can use the following syntax, which also helps when you need to pass props
`<Route path='/'><Home /></Route>`